### PR TITLE
daemon: add test for service

### PR DESCRIPTION
**Pull request by FeelyChau**
We have to downgrade typescript to 3.8.3 for mock. See:
https://github.com/microsoft/TypeScript/issues/39977

### CODE REVIEW

**Comment by FeelyChau** [comment]
Removed the cases for app service for now. At some point we need to resolve that we can't mock after TS 3.8.3.

File: README.md [hunk 1]
```diff
@@ -17,6 +17,8 @@
     <img alt="GitHub repo size" src="https://img.shields.io/github/repo-size/alibaba/pipcook"></a>
   <a href="https://opensource.org/licenses/Apache-2.0">
     <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg"></a>
+  <a href='https://coveralls.io/github/alibaba/pipcook'>
+    <img src='https://coveralls.io/repos/github/alibaba/pipcook/badge.svg' alt='Coverage Status' /></a>
```
Comment (yorkie) [inline README.md:21]: This needs to put in "Builds" section with tests

File: packages/daemon/src/service/app.ts [hunk 2]
```diff
@@ -44,6 +43,7 @@ export class AppService {
     // create pipelines
     await Promise.all(ctx.pipelines.map(async (pipeline) => {
       const data = await parseConfig(pipeline.config as RunConfigI);
+      console.log('pipeline.config', pipeline.config);
```
Comment (yorkie) [inline packages/daemon/src/service/app.ts:46]: Remove the log?

File: packages/daemon/src/service/pipeline.ts [hunk 3]
```diff
@@ -83,6 +84,7 @@ export class PipelineService {
   async removeJobById(id: string): Promise<number> {
     const job = await JobModel.getJobById(id);
     if (job) {
+      console.log('removeJobById', `${CoreConstants.PIPCOOK_RUN}/${job.id}`);
```
Comment (yorkie) [inline packages/daemon/src/service/pipeline.ts:87]: Ditto

File: packages/daemon/src/service/pipeline.ts [hunk 4]
```diff
@@ -286,21 +289,22 @@ export class PipelineService {
...
+      const runnable = this.runnableMap[id];
       if (runnable) {
         runnable.destroy();
-        delete runnableMap[id];
+        delete this.runnableMap[id];
       } else {
         console.error(`no runnable found: ${id}`);
       }
       job.status = PipelineStatus.CANCELED;
+      console.log(job);
```
Comment (yorkie) [inline packages/daemon/src/service/pipeline.ts:307]: Ditto.

**Comment by yorkie** [comment]
@FeelyChau Please have a double check your changes to make sure this is ready for merge.

**Review by yorkie — APPROVED** [review]

**Pull request merged**